hs_core/tasks.py

```diff
@@ -158,6 +168,69 @@ def nightly_periodic_task_check():
         file.write(timezone.now().strftime('%m/%d/%y %H:%M:%S'))
 
 
+@celery_app.task(ignore_result=True, base=HydroshareTask)
+def nightly_repair_resource_files():
+    """
+    Run repair_resource management command on resources updated in the last day
+    """
+    from hs_core.management.utils import check_time, repair_resource
+    start_time = time.time()
+    cuttoff_time = timezone.now() - timedelta(days=1)
+    recently_updated_resources = [res for res in BaseResource.objects.all() if res.last_updated >= cuttoff_time]
```

Comment:
I like the sound of that instead of list comprehension!
But... do you mean `last_updated`? (https://github.com/hydroshare/hydroshare/blob/2.8/hs_core/models.py#L2082)
If so, that's an @property so we can't use it in a filter...
Or am I missing `.updated` somewhere in our models?
Oh interesting, I see that we used `resource.updated` but I'm not seeing where that model field is defined. Help! :)

Reply:
The `updated `field is inherited from the `Page `class (mezzanine model).

Reply:
thank you, that explains it

Reply:
fixed